examples/builtins/mutatingwebhook.go

```diff
@@ -60,12 +60,3 @@ func (a *podAnnotator) Handle(ctx context.Context, req admission.Request) admiss
 
 	return admission.PatchResponseFromRaw(req.Object.Raw, marshaledPod)
 }
-
-// podAnnotator implements admission.DecoderInjector.
-// A decoder will be automatically injected.
-
-// InjectDecoder injects the decoder.
-func (a *podAnnotator) InjectDecoder(d *admission.Decoder) error {
```

Comment:
Can we update the `main.go` file to set the decoder for the two webhooks so the example actually works?